source/PowerAdjusterView.mc

```diff
@@ -137,6 +143,18 @@ class DataField extends Ui.SimpleDataField {
 
         if(info.currentPower != null) {
             avgPower = POWER_MULTIPLIER * SLOPE.interpolate(info.currentPower);
+
+            if (PURE_POWER) {
+                var PB = bikePower.getPedalPowerBalance();
```

Comment:
This is an instance of Toybox::AntPlus::PedalPowerBalance. You need to call pedalPowerPercent() and likely check rightPedalIndicator() to ensure it's a right-hand-side value.